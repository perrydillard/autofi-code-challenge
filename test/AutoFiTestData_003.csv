UUID,VIN,Model,Make,Mileage,Price,Year,Create Date
abc-123-456,JHM1239485KC,FJR1300,Yamaha,"45,000","4,000",2005,2020-06-01
def-789-098,TYT0943858DT,4Runner,Toyota,"13,754","20,000",2019,2020-03-02