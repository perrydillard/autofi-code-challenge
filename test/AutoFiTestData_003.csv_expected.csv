UUID,VIN,Make,Model,Mileage,Year,Price,Zip Code,Create Date,Update Date
abc-123-456,JHM1239485KC,Yamaha,FJR1300,"45,000",2005,"4,000",,2020-06-01,
def-789-098,TYT0943858DT,Toyota,4Runner,"13,754",2019,"20,000",,2020-03-02,
